### DISCUSSION TITLE
Unexpected motion of rigid body

### DISCUSSION THREAD

**Discussion by li2qn** [OpenRadioss/OpenRadioss#1753] [Q&A]
Hi Radioss experts

I have converted an ABAQUS example (compression of shell rings with general contact) to Radioss format to test the performance of Radioss explicit solver. The converted model runs fine, however there are some issues I have noticed in the model. I would like to looking for answers from Radioss experts here. Any suggestion/comments would be appreciated. 

Simulated problem as shown below, the lid and box are both modeled as rigid body. The box fixed in all DOF and the Lid only allowed to move in y direction with constant velocity BC 
![image](https://github.com/OpenRadioss/OpenRadioss/assets/48695377/a358b948-844c-49af-9e50-c6a5d37756bf)
![image](https://github.com/OpenRadioss/OpenRadioss/assets/48695377/240b94c8-d5c8-45ea-b6b0-f78e9ae2cf05)

Issue:

1) I have noticed unexpected movement of the lateral movement of both Right Lid and Box which should be only allowed to move in y and completely fixed in all DOFs separately.
2) The reaction forces in y direction are quite different compare to ABAQUS results. 
3) The deformation of the shell rings are also quite different.

I am new to Radioss, please help me check what's missing in my Radioss Model. Thanks in advance. I have updated the abaqus input file and converted Radioss format as well as some of the results (abaqus Left, Radioss Right).

![image](https://github.com/OpenRadioss/OpenRadioss/assets/48695377/b28fd358-a03c-4618-a29d-2e9163011ab2)
![image](https://github.com/OpenRadioss/OpenRadioss/assets/48695377/5832c621-8897-4b95-a0b7-eb3aaaaf19d4)

[abaqus.zip](https://github.com/OpenRadioss/OpenRadioss/files/12235264/abaqus.zip)
[Radioss.zip](https://github.com/OpenRadioss/OpenRadioss/files/12235265/Radioss.zip)

**Comment by hotaosa** [comment]
Hello, li2qn

I checked your radioss input and found some issues. I referred to Radioss2022 Reference Guide.

1. The degrees of freedom in /NBCS need to have a space between translational and rotational degree of freedom. Setting it to '111 111' will eliminate box movement.
2. Iflag in /MAT/LAW2 needs to be written in Column 5.
3. The reason for the difference in deformation by Abaqus is not clear for me, but changing the friction coefficient (to 0.1) results closer to Abaqus.

I hope it helps you. If you notice any mistakes, please let me know.

**Comment by li2qn** [comment]
Thanks so much for your investigation. Could you please upload the files that you modified that able to eliminate the box movement? IMO, 101111 and 101 111 does the same thing. Maybe I am not right. And what value of friction are you given to reach the closer deformation? Thanks for your input again. BestLeeOn Aug 3, 2023, at 2:52 PM, hotaosa ***@***.***> wrote:﻿
Hello, li2qn
I checked your radioss input and found some issues. I referred to Radioss2022 Reference Guide.

The degrees of freedom in /NBCS need to have a space between translational and rotational degree of freedom. Setting it to '101 111' will eliminate box movement.
Iflag in /MAT/LAW2 needs to be written in Column 5.
The reason for the difference in deformation by Abaqus is not clear for me, but changing the friction coefficient results closer to Abaqus.

I hope it helps you. If you notice any mistakes, please let me know.

**Comment by hotaosa** [comment]
Thanks for your reply. I will upload the input files and capture of deformation when the friction coefficient was set to 0.1.
[shell_compact_f_radioss.zip](https://github.com/OpenRadioss/OpenRadioss/files/12248692/shell_compact_f_radioss.zip)

In my environment (Windows, OpenRadioss built on May 29, 2023), without spaces in /NBCS, OpenRadioss starter gave me a warning of 'INCOMPATIBLE KINEMATIC CONDITIONS'.

**Reply by MarianBulla** [reply]
Hi hotaosa, 
thank you for checked your model. 
There are some initial penetrations within this model. 

After a small correction of these few nodes, it runs without any WARNING nor ERROR till the end. (Model is attached, here)

Hope it helps you ?

Thanks and best regards,
Marian
[mod.zip](https://github.com/OpenRadioss/OpenRadioss/files/12250859/mod.zip)

P.S.: I'm using Windows 11, OpenRadioss built today, 03 Aug. 2023.

**Comment by li2qn** [comment]
﻿Hi Marian,Thanks for your time and investigation of the model again. May I ask if the reaction force results comparable to ABAQUS? I am still not clear what’s causing the difference. Thanks for your help again.BestLeeSent from my iPhoneOn Aug 3, 2023, at 8:38 PM, Marian Bulla ***@***.***> wrote:﻿
Hi hotaosa,
thank you for checked your model.
There are some initial penetrations within this model.
After a small correction of these few nodes, it runs without any WARNING nor ERROR till the end. (Model is attached, here)
Hope it helps you ?
Thanks and best regards,
Marian
mod.zip
P.S.: I'm using Windows 11, OpenRadioss built today, 03 Aug. 2023.

**Comment by li2qn** [comment]
﻿﻿Hi Marian,Thanks for your time and investigation of the model again. May I ask if the reaction force results comparable to ABAQUS? I am still not clear what’s causing the difference. Thanks for your help again.BestLeeSent from my iPhoneOn Aug 3, 2023, at 8:38 PM, Marian Bulla ***@***.***> wrote:﻿
Hi hotaosa,
thank you for checked your model.
There are some initial penetrations within this model.
After a small correction of these few nodes, it runs without any WARNING nor ERROR till the end. (Model is attached, here)
Hope it helps you ?
Thanks and best regards,
Marian
mod.zip
P.S.: I'm using Windows 11, OpenRadioss built today, 03 Aug. 2023.
